.github/workflows/parachain.yml

```
@@ -3,21 +3,21 @@ name: parachain
 on:
   push:
     paths:
-      - 'parachain/**'
-      - '!parachain/README.md'
-      - '!parachain/LICENSE'
+      - "parachain/**"
+      - "!parachain/README.md"
+      - "!parachain/LICENSE"
     branches:
       - main
   pull_request:
     paths:
-      - 'parachain/**'
-      - '!parachain/README.md'
-      - '!parachain/LICENSE'
+      - "parachain/**"
+      - "!parachain/README.md"
+      - "!parachain/LICENSE"
   workflow_dispatch:
 
 jobs:
   fmt:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

I've setup a [larger github-hosted runner](https://docs.github.com/en/actions/using-github-hosted-runners/using-larger-runners) for our org, with 8 vcpus and 32GiB of RAM. Should help a lot to decreases execution time.

To use it:

```suggestion
    runs-on: snowbridge-runner
```